test/parallel/test-fs-realpath.js

```diff
@@ -225,7 +225,7 @@ function test_cyclic_link_protection(callback) {
   });
   assert.throws(() => {
     fs.realpathSync(entry);
-  }, common.expectsError({ code: 'ELOOP', type: Error }));
+  }, /^Error: ELOOP: too many symbolic links encountered, stat/);
```

Comment:
I'm -1 on changing this one. The check here is correct, tho the message check can be added.